qiita_ware/private_plugin.py

```diff
@@ -291,6 +291,14 @@ def delete_sample_or_column(job):
         job._set_status('success')
 
 
+def _delete_analysis_artifacts(analysis):
+    aids = [a.id for a in analysis.artifacts]
```

Comment:
The lines of code that this function is replacing also check for `if a.processing_parameters is None` and also delete `a.descendants.nodes()`. Why are these no longer relevant? Did something else change in the code ever since that was written?

Reply:
Yup, no needed anymore; I'm actually not sure why we decided to delete that way - it makes no sense to me today.

Reply:
Gotcha, thanks!